src/styles/Home.module.css

```diff
@@ -1,48 +1,162 @@
-  z-index: 0;
-  background-size: cover;
-  height: 500px;
-  top: 0;
+  width: 100%;
+  z-index: 1;
+}
+
+.flex {
+  z-index: 2;
+  display: flex;
+  flex-direction: row;
+  background-color: #000000;
+  position: relative;
+  border-radius: 20px;
+  margin: 0 36px 0 380px;
+  max-width: 600px;
+  margin:0px auto 0 auto;
+  transform: translateY(-500%);
+  align-items: center;
+  justify-items: center;
+  width: fit-content;
+  height:5rem
+
 }
 
 .landingText {
-  position: absolute;
-  z-index: 0;
-  background-color: black;
+  font-family: 'Mukta', sans-serif;
+  font-style: normal;
+  font-size: 44px;
+  line-height: 100%;
   color: white;
-  text-align: left;
-  align-items: left;
-  margin-top: 200px;
-  margin-left: 100px;
-  top: 0;
+  padding: 27px 10px 29px 33px;
+  position: relative;
+  text-align: center;
+  font-weight: 600;
+  
+}
+
+.landingArrow {
+  position: relative;
+  top: 35%;
+  right: 2px;
+  width: 30px;
+  padding-right: 4px;
+  transform: translateY(-80%);
+}
+
+.vector {
+  z-index: 2;
+  align-items: center;
+  padding: 20px;
+  transform: translateY(-110%);
+  margin-left: 50%;
+  width: 10%;
 }
 
 .landingIntro {
-  margin: 20px 40px;
-  padding: 5px 2px;
-  text-align: left;
+  position: relative;
+  margin: -20% 160px 20px 115px;
+  
+  left: calc(50% - 800.05px / 2 - 204.98px);
+  letter-spacing: -0.02em;
 }
 
 .paragraph {
-  margin: 10px 40px;
+  margin: 0px 160px 0px 115px;
   padding: 5px 2px;
-  text-align: left;
+  font-family: 'Montserrat', sans-serif;
+  color: #080d26;
+
+  font-style: normal;
+  font-size: 20px;
+  line-height: 140%;
+  text-align: justify;
+}
+
+.button{
+  background-color: #1B2B7B;
+  align-content: 'center';
+  font-family: 'Montserrat',sans-serif;
+  border-radius: 30px;
+  margin: 7px 5% 7px 0px;
+  width: 194px;
+  height: 40px;
+  font-size: 16px;
+  font-weight: 500;
+  box-shadow: none;
+  }
+
+.button:hover {
+  background-color: #1B2B7B;
+  color: 'white'
 }
 
-.card {
-  margin-top: 60px;
-  margin-bottom: 60px;
-  margin-left: 40px;
-  margin-right: 40px;
+@media (max-width: 1025px) {
+  .landingIntro {
+    margin: -320px 56px 0px 33px;
+  }
+  .home{
+    margin:0px
+  }
+  .paragraph,
+  .card {
+    margin: 0px 56px 0px 33px;
+  }
+  .flex {
+    transform: translateY(-350%);
+    height:10vh;
+  }
+  .landingText {
+    line-height: 100%;
+    font-size: 30px;
+    color: white;
+    padding: 0px 27px 27px 48px; 
+    position: relative;  
+  }
+  .landingArrow {
+    position: relative;
+    top: 35%;
+    right: 2px;
+    width: 30px;
+    flex-shrink: 0;
+    padding-right: 4px;
+  }
+
+}
+
+@media (max-width: 377px) {
+  .vector {
+    visibility: hidden;
+  }
+  
+  .home{
+    margin:0px
+  }
+
+  .flex {
+    margin: 20% auto -20% auto;
+    transform: translateY(-300%);
+    width: 70vw;
+    height:10vh
+  
+  }
+  .landingText {
+    font-size: 24px;
+    line-height: 100%;
+    color: white;
+    padding: 25px 10px 27px 48px;
+    font-weight: 500;
+  }
```

Comment:
As I said, We are migrating styles from CSS modules to CSS-in-JS so you can convert the styles you have added in this file using the [sx prop](https://mui.com/system/the-sx-prop/) and [styled()](https://mui.com/system/styled/) that MUI provides and can add those to `src/pages/index.js`